pipeline/mist.go

```diff
@@ -36,10 +38,17 @@ func (m *mist) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	job.SegmentingTargetURL = segmentingTargetURL.String()
 	log.AddContext(job.RequestID, "segmented_url", job.SegmentingTargetURL)
 
+	if !isVideo(job.RequestID, job.SourceFile) {
+		return nil, fmt.Errorf("source was not a video: %s", job.SourceFile)
```

Comment:
does anyone know offhand how this error would end up appearing to the user? do we have a special way to signal "bad request" type errors to the user?

Reply:
Not at the moment, this will just show up in logs